api/RTCPeerConnection.json

```diff
@@ -244,57 +244,43 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/RTCPeerConnection/connectionState",
           "support": {
             "webview_android": {
-              "version_added": "56"
+              "version_added": null
```

Comment:
This should be `false`, as should the all Chrome and Opera flavors. There is an [open Chrome ticket](https://bugs.chromium.org/p/chromium/issues/detail?id=823144) on this.

Reply:
needs a lot more false then, will change (and link chrome and firefox bugs in the commit message)

Reply:
squashed, pushed and added link to https://bugzilla.mozilla.org/1265827